standards/production-readiness.md

```diff
@@ -0,0 +1,22 @@
+# Production Readiness
+
+Before launching a new service or application to production, it SHOULD be reviewed by the team to ensure production readiness and adherence with engineering standards.
+
+A pre-launch checklist SHOULD be completed and can be completed in a pre-launch team meeting:
+
+![Production Readiness Whiteboard](../images/production-readiness.jpg)
+
+## Sample Pre-Launch Checklist
+
+| Category                                                                                                                                                                                                                                                   | Completeness (1-10) |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| Documentation [1](https://github.com/NYPL/engineering-general/blob/master/standards/documentation.md)                                                                                                                                                      |                     |
+| Peer Review [1](https://github.com/NYPL/engineering-general/blob/master/standards/peer-review.md)                                                                                                                                                          |                     |
+| Logging [1](https://github.com/NYPL/engineering-general/blob/master/standards/logging.md)                                                                                                                                                                  |                     |
+| Deployment [1](https://github.com/NYPL/engineering-general/blob/master/standards/node-lambda.md)                                                                                                                                                           |                     |
+| Coding Standards [1](https://github.com/NYPL/engineering-general/blob/master/standards/coding-standards.md), [2](https://github.com/NYPL/engineering-general/blob/master/standards/naming-conventions.md)                                                  |                     |
+| Testing [1](https://github.com/NYPL/engineering-general/blob/master/standards/test-coverage.md)                                                                                                                                                            |                     |
+| Accessibility [1](https://github.com/NYPL/engineering-general/blob/master/standards/accessibility.md)                                                                                                                                                      |                     |
```

Comment:
Great addition to our standards @kfriedman -- I would note that some categories may not be required. For example, the CircTransPoller is a back-end service where `accessibility` would not be applied.

Reply:
Great suggestion. I added a note!